docs/root/configuration/operations/tools/router_check.rst

```diff
@@ -183,3 +183,20 @@ fields that could be tested.
   > bazel-bin/test/tools/router_check/router_check_tool --config-path ... --test-path ... --useproto --f 7 --covall
   Current route coverage: 6.2948%
   Failed to meet coverage requirement: 7%
+
+When the fail under flag is set an optional flag (`-i` or `--incr-cov-fail`) can
+be included to enforce the specified fail under coverage threshold matches the current coverage.
+This prevents the specified fail under test coverage value from becoming stale as new tests are added
+and ensures coverage regressions don't occur if the value is set below the current coverage.
+
+.. note::
+
+  This will only take effect if the fail under flag is set and the current coverage
+  requirement is met.
+
+.. code:: bash
+
+  > bazel-bin/test/tools/router_check/router_check_tool --config-path ... --test-path ... --useproto --f 7 -i --covall
+  Current route coverage: 6.2948%
+  Failed due to stale coverage failure threshold.
+  Please increase the coverage failure threshold to match existing coverage of 89.4231%
```

Comment:
I don't think this makes sense? Should it be `-f 89.4231` or something? Isn't the idea here that the coverage was actually increased to the user is supposed to then go back and increase the value being passed to `-f`?

This still only vaguely makes sense to me. Can you potentially have the documentation be more clear about the suggestion of how this flag would be used in a real testing pipeline?